### PR TITLE
Edit techbutton UI and minimap button

### PR DESCRIPTION
**Pull request by lishaoxia1985**
1. techbutton:
YueR
2. minimap button, look at the cattle button:
XueR

### CODE REVIEW

**Comment by yairm210** [comment]
Completely agree with the cattle button
As for the tech button, I'm not sure. I think we'll put a pool on Discord.
Could you send me a 'before' picture for comparison?

**Comment by lishaoxia1985** [comment]
YuneR
- you should concern about techbutton in the worldscreen.
- In the MainMenuScreen using the Rounded rectangle as the background of button is not a good choice.
- Not like rectangle, sometimes the Rounded rectangle as the background will be out of shape.

**Comment by lishaoxia1985** [comment]
Fenger
@yairm210 besides the language Russian the tech button will show very well.
maybe we can move the label about tech name on the top of the button and make it have a separate line.

File: core/src/com/unciv/ui/pickerscreens/TechButton.kt [hunk 1]
```diff
@@ -69,8 +69,6 @@ class TechButton(techName:String, private val techManager: TechManager, isWorldS
                     .apply { color = Color.BLACK }.surroundWithCircle(techIconSize))
 
         if (isWorldScreen) rightSide.add(techEnabledIcons)
-        else rightSide.add(techEnabledIcons)
-                .width(techEnabledIcons.children.size * (techIconSize+6f))
-                .minWidth(150f)
+        else rightSide.add(techEnabledIcons).width(225f)
```
Comment (lishaoxia1985) [inline core/src/com/unciv/ui/pickerscreens/TechButton.kt:72]: if you want that Russian language shows very well, please make the width number larger, but I'm not sure it's good or bad.

**Comment by yairm210** [comment]
The poll seems to indicate that the round tech buttons are definitely better loved

**Comment by yairm210** [comment]
Added the resource toggle in a separate commit

**Pull request closed without merging**